### ISSUE TITLE
restore older versions that were on dockerhub under nginxdemos/nginx-ingress

### ISSUE THREAD

**Issue by bw2** [nginxinc/kubernetes-ingress#361]
**Describe the bug**

The nginxdemos/nginx-ingress:0.8.0 image was removed from docker hub and broke our builds.
It was availabe until recently.
https://webcache.googleusercontent.com/search?q=cache:UxwG2TKY2WIJ:https://hub.docker.com/r/nginxdemos/nginx-ingress/tags/+&cd=1&hl=en&ct=clnk&gl=us

Could you please restore that version to https://hub.docker.com/r/nginx/nginx-ingress/tags/ ?

**Comment by pleshakov** [comment]
We recently removed the nginxdemos/nginx-ingress repo after we had stopped publishing new releases in it in January 2018. Apologies for the inconvenience. 

0.8.0 is an old version from April 2017. Since that release, the Ingress Controller was significantly improved: new features added, several bugs fixes, stability/performance issues addressed. We highly recommend you to upgrade to the latest stable release - 1.3.0.



I responded to your issue https://github.com/nginxinc/kubernetes-ingress/issues/360 and hope we can quickly solve the upgrade problem.
